proto/proto.go

```diff
@@ -252,6 +254,53 @@ func (j Jobs) Swap(i, k int) {
 	j[i], j[k] = j[k], j[i]
 }
 
+// RequestFilter represents optional filters when listing requests.
+type RequestFilter struct {
+	Type      string // Type of requests to return.
+	States    []byte // Request states to include.
+	Requestor string // User who made the request.
```

Comment:
The return data is `user` (and column name in db the same). Any reason it's `Requestor` here? (Also, code comment says "User who...".